directory/datadog_checks/directory/directory.py

```diff
@@ -8,6 +8,7 @@
 
 from datadog_checks.checks import AgentCheck
 from datadog_checks.config import is_affirmative
+from datadog_checks.errors import ConfigurationError
```

Comment:
this exception only exists since https://github.com/DataDog/integrations-core/pull/2367, shouldn't this be reflected on the version constraint of this integration on the `datadog-checks-base` wheel? (just asking, not sure what the policy is)

Reply:
We're still transitioning and very few of the checks pin the base check version, `directory` is one of those not pinning. At this moment there's not a real policy, we help users who need to update to manually setup the right set of packages in the agent on a case by case policy. This will change as soon as we outline the Agent CLI requirements for the single check update.